app/src/main/java/com/example/hungrywolfes/ui/detailsScreen/DetailsViewModel.kt

```diff
@@ -20,12 +19,20 @@ class DetailsViewModel : ViewModel() {
     private val _detailsMeal = MutableLiveData<ListMealsDetails>()
     val detailsMeal: LiveData<ListMealsDetails> = _detailsMeal
 
-    val stringTags =_detailsMeal.map {
+    val stringTags = _detailsMeal.map {
         it.strTags?.split(",")
     }
 
-    private val _addItemToFavoriteButton = MutableLiveData(false)
-    val addItemToFavoriteButton: LiveData<Boolean> = _addItemToFavoriteButton
+    val buttonCheckStatus: MutableLiveData<Boolean> = MutableLiveData<Boolean>()
+
+    private val _favoriteMealList = MutableLiveData<MutableList<ListMealsImages>>()
```

Comment:
There's no need to use a liveData here since you don't display this list anywhere in the details screen, a private list will suffice.